app/controllers/post_types_controller.rb

```diff
@@ -5,8 +5,8 @@ class PostTypesController < ApplicationController
   def create
     post_type_params = params.require(:post_type).permit(:name, :group_id)
     post_type = PostType.create(post_type_params)
-    return redirect_to :back unless post_type.errors.any?
-    redirect_to :back, alert: post_type.errors.to_a.first
+    return redirect_back unless post_type.errors.any?
+    redirect_back alert: post_type.errors.to_a.first
```

Comment:
I think we should add group url as fallback location for these two, because it is a step closer to the post giving page, than the root url.